tests/apikey-atomic-bindings.test.ts

```
@@ -0,0 +1,346 @@
+            org_id: TEST_ORG_ID,
+          },
+        ],
+      }),
+    })
+
+    const data = await response.json() as { error: string }
+    expect(response.status).toBe(400)
+    expect(data.error).toBe('invalid_bindings')
+  })
+
+  it('RBAC-only key (mode=NULL) can authenticate', async () => {
```

ESLint

[error] `it`s should begin with lowercase

(test/prefer-lowercase-title)